internal/usecase/client/handshake.go

```
@@ -83,11 +83,11 @@ func watchHandshakeTransport(ctx context.Context, transport ports.Transport) fun
 	}
 }
 
-func boundedHandshakeOperation(ctx context.Context, transport ports.Transport, operation func() error) error {
+func boundedHandshakeOperation(ctx context.Context, transport ports.ClientConnection, operation func() error) error {
```

golangci-lint (2.12.2)

[error] func boundedHandshakeOperation is unused

(unused)